### DISCUSSION TITLE
About RSSI signal strength

### DISCUSSION THREAD

**Discussion by etmomoabc** [mKeRix/room-assistant#1172] [General]
I've been using RA for almost a year and it's always been stable. But since a month ago (not sure what the upgrade was), the distance measurement is very unstable. Same distance, the values jump from 0.6-1 to 7-9. Interesting is that its value at 0.6-1 can be kept for a few days, and then 7-9 can be kept for a few days.

**Comment by DaWheelz** [comment]
I have been experiencing the exact same thing, good to know I'm not the only person with this problem. Hope we will get a solution soon or it will be fixed.
